src/library/tabledelegates/stardelegate.cpp

```diff
@@ -68,24 +91,111 @@ void StarDelegate::setModelData(QWidget* editor, QAbstractItemModel* model,
 void StarDelegate::commitAndCloseEditor() {
     StarEditor* editor = qobject_cast<StarEditor*>(sender());
     emit commitData(editor);
-    emit closeEditor(editor);
+    emit closeEditor(editor, QAbstractItemDelegate::SubmitModelCache);
+}
+
+void StarDelegate::closingEditor(QWidget* widget, QAbstractItemDelegate::EndEditHint hint) {
+    Q_UNUSED(hint);
```

Comment:
We've recently updated our [coding guideline](https://github.com/mixxxdj/mixxx/wiki/Coding-Guidelines#unused-parameters) to the unnamed approach

(Nit: rename `widget` to `pWidget`)

```suggestion
void StarDelegate::closingEditor(QWidget* pWidget, QAbstractItemDelegate::EndEditHint) {
```